source/_integrations/waze_travel_time.markdown

```diff
@@ -20,7 +20,7 @@ The `waze_travel_time` sensor provides travel time from the [Waze](https://www.w
 Notes:
 
 - If a unit system is not specified, the integration will use the unit system configured on your Home Assistant instance.
-- Origin and Destination can be the address or the GPS coordinates of the location (GPS coordinates has to be separated by a comma). You can also enter an entity id which provides this information in its state, an entity id with latitude and longitude attributes, or zone friendly name (case sensitive).
+- Origin and Destination can be the address or the GPS coordinates of the location (GPS coordinates has to be separated by a comma). You can also enter an entity id which provides this information in its state, an entity id with latitude and longitude attributes, or zone friendly name (case sensitive). For origin, most people will want to use the device that tracks their location. I.e device_tracker.kevin_pixel_gps.
```

Comment:
Since you opened a PR, also id should be ID to be consistent throughout the document since I now see both used.